### PR TITLE
Added the advanced movements to the arm

### PR DESCRIPTION
**Pull request by ChielBruin**
# ~~First merge #36~~
- A straight grasping movement
- A straight movement back when gripping fails
- Wrist tilting of +- 15 degrees when the vacuum cannot be created

This can be tested with the fllowing command:
```
rosservice call /target/cucumber "cucumber:
  header:
    seq: 0
    stamp: {secs: 0, nsecs: 0}
    frame_id: ''
  dimension: 3
  stem_position: {x: -0.00, y: 0.600, z: 0.30}
  width: 0.0
  height: 0.0
  image_stem_position: [0,0]
  curvature: 0.0
dropLocation:
  position: {x: 0.0, y: 0.0, z: 0.0}
  orientation: {x: 0.0, y: 0.0, z: 0.0, w: 0.0}"
```

NOTE: You should set pin DI3 to 1 to be able to see the gripper failing routine

### CODE REVIEW

File: catkin_ws/src/kpr_arm_control/src/py_arm_control/states.py [hunk 1]
```diff
@@ -40,17 +41,28 @@ def execute(self, userdata):
 		@return 'MoveOK' when the move was successful, 'MoveError' otherwise
 		'''
 		rospy.loginfo('Executing state MoveToCucumber')
-		pose = Pose(userdata.data.cucumber.stem_position, Quaternion(0,0,0,1))
-		# TODO: Make the last section of movement straight towards the produce
+<<<<<<< HEAD
```
Comment (MaxGroenenboom) [inline catkin_ws/src/kpr_arm_control/src/py_arm_control/states.py]: I think you missed this conflict line

File: catkin_ws/src/kpr_arm_control/src/py_arm_control/states.py [hunk 2]
```diff
@@ -40,17 +41,27 @@ def execute(self, userdata):
...
-			userdata.result.moveToTarget = HarvestStatus(success = HarvestStatus.ERROR, message = 'Error planning the movement')
+			userdata.result.moveToTarget = HarvestStatus(success = HarvestStatus.ERROR, message = 'Error planning the movement to the grasp start position')
 			return 'MoveError'
 		elif res is MoveStatus.MOVE_OK:
-			userdata.result.moveToTarget = HarvestStatus(success = HarvestStatus.OK, message = 'Success')
-			return 'MoveOK'
+			pose.position.y = pose.position.y + 0.1
+			res = self.moveArmTo(pose)
+			if res is MoveStatus.PLAN_ERROR:
+				userdata.result.moveToTarget = HarvestStatus(success = HarvestStatus.ERROR, message = 'Error planning straight the movement to the produce')
```
Comment (MaxGroenenboom) [inline catkin_ws/src/kpr_arm_control/src/py_arm_control/states.py]: This sentence is a bit weird

Comment (ChielBruin) [reply catkin_ws/src/kpr_arm_control/src/py_arm_control/states.py]: You're right that needs some clarification

**Review by MaxGroenenboom — APPROVED** [review]
Lookin' good

**Pull request merged**